infra/main.bicep

```diff
@@ -286,62 +381,62 @@ module backend 'core/host/appservice.bicep' = {
-      // Shared by all OpenAI deployments
-      OPENAI_HOST: openAiHost
-      AZURE_OPENAI_EMB_MODEL_NAME: embedding.modelName
-      AZURE_OPENAI_EMB_DIMENSIONS: embedding.dimensions
-      AZURE_OPENAI_CHATGPT_MODEL: chatGpt.modelName
-      AZURE_OPENAI_GPT4V_MODEL: gpt4vModelName
-      // Specific to Azure OpenAI
-      AZURE_OPENAI_SERVICE: isAzureOpenAiHost && deployAzureOpenAi ? openAi.outputs.name : ''
-      AZURE_OPENAI_CHATGPT_DEPLOYMENT: chatGpt.deploymentName
-      AZURE_OPENAI_EMB_DEPLOYMENT: embedding.deploymentName
-      AZURE_OPENAI_GPT4V_DEPLOYMENT: useGPT4V ? gpt4vDeploymentName : ''
-      AZURE_OPENAI_API_VERSION: azureOpenAiApiVersion
-      AZURE_OPENAI_API_KEY_OVERRIDE: azureOpenAiApiKey
-      AZURE_OPENAI_CUSTOM_URL: azureOpenAiCustomUrl
-      // Used only with non-Azure OpenAI deployments
-      OPENAI_API_KEY: openAiApiKey
-      OPENAI_ORGANIZATION: openAiApiOrganization
-      // Optional login and document level access control system
-      AZURE_USE_AUTHENTICATION: useAuthentication
-      AZURE_ENFORCE_ACCESS_CONTROL: enforceAccessControl
-      AZURE_ENABLE_GLOBAL_DOCUMENT_ACCESS: enableGlobalDocuments
-      AZURE_ENABLE_UNAUTHENTICATED_ACCESS: enableUnauthenticatedAccess
-      AZURE_SERVER_APP_ID: serverAppId
-      AZURE_SERVER_APP_SECRET: serverAppSecret
-      AZURE_CLIENT_APP_ID: clientAppId
-      AZURE_CLIENT_APP_SECRET: clientAppSecret
-      AZURE_TENANT_ID: tenantId
-      AZURE_AUTH_TENANT_ID: tenantIdForAuth
-      AZURE_AUTHENTICATION_ISSUER_URI: authenticationIssuerUri
-      // CORS support, for frontends on other hosts
-      ALLOWED_ORIGIN: allowedOrigin
-      USE_VECTORS: useVectors
-      USE_GPT4V: useGPT4V
-      USE_USER_UPLOAD: useUserUpload
-      AZURE_USERSTORAGE_ACCOUNT: useUserUpload ? userStorage.outputs.name : ''
-      AZURE_USERSTORAGE_CONTAINER: useUserUpload ? userStorageContainerName : ''
-      AZURE_DOCUMENTINTELLIGENCE_SERVICE: documentIntelligence.outputs.name
-      USE_LOCAL_PDF_PARSER: useLocalPdfParser
-      USE_LOCAL_HTML_PARSER: useLocalHtmlParser
-    }
+    appSettings: appEnvVariables
+  }
+}
+
+// Azure container apps resources
+
+// identity for pulling images from ACR
+module acaIdentity 'core/security/aca-identity.bicep' = if (deploymentTarget == 'containerapps') {
+  name: 'aca-identity'
+  scope: resourceGroup
+  params: {
+    identityName: acaIdentityName
+    location: location
+  }
+}
+
+module containerApps 'core/host/container-apps.bicep' = if (deploymentTarget == 'containerapps') {
+  name: 'container-apps'
+  scope: resourceGroup
+  params: {
+    name: 'app'
+    tags: tags
+    location: location
+    workloadProfile: azureContainerAppsWorkloadProfile
+    containerAppsEnvironmentName: acaManagedEnvironmentName
+    containerRegistryName: '${containerRegistryName}${resourceToken}'
+    logAnalyticsWorkspaceResourceId: monitoring.outputs.logAnalyticsWorkspaceId
+  }
+}
+
+// The application frontend
```

Comment:
This comment says "frontend" but it's called backend. Since it's a SPA maybe we just say application?